### ISSUE TITLE
Release 2.3.1 has broken POM file - no dependencies declared

### ISSUE THREAD

**Issue by vyazelenko** [mockito/mockito#816]
Release `2.3.0` has the following info in [mockito-core-2.3.0.pom](http://jcenter.bintray.com/org/mockito/mockito-core/2.3.0/:mockito-core-2.3.0.pom):
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>org.mockito</groupId>
  <artifactId>mockito-core</artifactId>
  <version>2.3.0</version>
  <dependencies>
    <dependency>
      <groupId>net.bytebuddy</groupId>
      <artifactId>byte-buddy</artifactId>
      <version>1.5.5</version>
      <scope>runtime</scope>
    </dependency>
    <dependency>
      <groupId>net.bytebuddy</groupId>
      <artifactId>byte-buddy-agent</artifactId>
      <version>1.5.5</version>
      <scope>runtime</scope>
    </dependency>
    <dependency>
      <groupId>org.objenesis</groupId>
      <artifactId>objenesis</artifactId>
      <version>2.4</version>
      <scope>runtime</scope>
    </dependency>
  </dependencies>
  <name>Mockito</name>
  <packaging>jar</packaging>
  <url>http://mockito.org</url>
  <description>Mock objects library for java</description>
  <licenses>
    <license>
      <name>The MIT License</name>
      <url>http://github.com/mockito/mockito/blob/master/LICENSE</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <scm>
    <url>http://github.com/mockito/mockito</url>
  </scm>
  <issueManagement>
    <url>https://github.com/mockito/mockito/issues</url>
    <system>GitHub issues</system>
  </issueManagement>
  <ciManagement>
    <url>https://travis-ci.org/mockito/mockito</url>
    <system>TravisCI</system>
  </ciManagement>
  <developers>
    <developer>
      <id>szczepiq</id>
      <name>Szczepan Faber</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
    <developer>
      <id>bric3</id>
      <name>Brice Dutheil</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
    <developer>
      <id>raphw</id>
      <name>Rafael Winterhalter</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
    <developer>
      <id>TimvdLippe</id>
      <name>Tim van der Lippe</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
  </developers>
</project>
```
However release `2.3.1` has only the following info in [mockito-core-2.3.1.pom](http://jcenter.bintray.com/org/mockito/mockito-core/2.3.1/:mockito-core-2.3.1.pom):
```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>org.mockito</groupId>
  <artifactId>mockito-core</artifactId>
  <version>2.3.1</version>
  <name>mockito-core</name>
  <packaging>jar</packaging>
  <url>http://mockito.org</url>
  <description>Mockito mock objects library core API and implementation</description>
  <licenses>
    <license>
      <name>The MIT License</name>
      <url>http://github.com/mockito/mockito/blob/master/LICENSE</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
  <scm>
    <url>http://github.com/mockito/mockito</url>
  </scm>
  <issueManagement>
    <url>https://github.com/mockito/mockito/issues</url>
    <system>GitHub issues</system>
  </issueManagement>
  <ciManagement>
    <url>https://travis-ci.org/mockito/mockito</url>
    <system>TravisCI</system>
  </ciManagement>
  <developers>
    <developer>
      <id>szczepiq</id>
      <name>Szczepan Faber</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
    <developer>
      <id>bric3</id>
      <name>Brice Dutheil</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
    <developer>
      <id>raphw</id>
      <name>Rafael Winterhalter</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
    <developer>
      <id>TimvdLippe</id>
      <name>Tim van der Lippe</name>
      <roles>
        <role>Core developer</role>
      </roles>
    </developer>
  </developers>
</project>
```

**Comment by TimvdLippe** [comment]
Hm, version `2.3.1` should not be available as the last published version is `2.3.0` according to our git history: https://github.com/mockito/mockito/commits/release/2.x

@szczepiq I think #809 broke the release mechanism and is publishing artifacts in pullrequests

**Comment by bric3** [comment]
I'm closing as it should be resolved by #809.

Thanks @vyazelenko for the vigilante update, I will unpublish these artifacts 2.3.1 to avoid further havoc.

**Comment by szczepiq** [comment]
Thank you very much for reporting! I will double check this behavior today.

Thanks @bric3 for unpublishing the artifacts!

**Comment by vyazelenko** [comment]
Well, I'm using [gradle-versions-plugin](https://github.com/ben-manes/gradle-versions-plugin) to check if new versions are available and it picked up `2.3.1` from Bintray. 
Thanks for quick fix.
